go/pkg/pwapi/api_user-get-session.go

```diff
@@ -95,10 +95,18 @@ func (svc *service) loadUserSeasons(ctx context.Context) ([]*UserGetSession_Outp
 		return nil, errcode.ErrGetUserOrganizations.Wrap(err)
 	}
 
+	req := svc.db
+	if isTesterContext(ctx) {
+		req = req.
+			Where(pwdb.Season{Visibility: pwdb.Season_Public}).
+			Or(pwdb.Season{IsTesting: true})
+	} else if !isAdminContext(ctx) {
+		req = req.
+			Where(pwdb.Season{Visibility: pwdb.Season_Public})
+	}
```

Comment:
can you just rewrite this with a switch case to make it less painful to read?

```go
switch {
    case isAdmin: // because it's the highest level
      // noop
    case isTester:
      req = req.Where(tester)
    default: // "normal" user
      req = req.Where(normal)
}
```

thanks